cluster-service/deploy/templates/k8s-job-arbitrary-commands.yaml

```diff
@@ -0,0 +1,103 @@
+{{ if .Values.deployDebugJob }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cs-arbitrary-commands
+  namespace: '{{ .Release.Namespace }}'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: cs-arbitrary-commands-role
+  namespace: '{{ .Release.Namespace }}'
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
+    verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cs-arbitrary-commands-rolebinding
+  namespace: '{{ .Release.Namespace }}'
+subjects:
+  - kind: ServiceAccount
+    name: cs-arbitrary-commands
+    namespace: '{{ .Release.Namespace }}'
+roleRef:
+  kind: Role
+  name: cs-arbitrary-commands-role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cs-arbitrary-commands
+  namespace: '{{ .Release.Namespace }}'
+spec:
+  ttlSecondsAfterFinished: 300
+  activeDeadlineSeconds: 900
+  template:
+    spec:
+      serviceAccountName: cs-arbitrary-commands
+      restartPolicy: Never
+      containers:
+      - name: cs-arbitrary-commands
+        image: 'mcr.microsoft.com/aks/command/runtime:master.240118.1'
```

Comment:
How did we decide to use this image? what is the level of freedom we have available around the base image?